tests/test.py

```diff
@@ -23,19 +23,15 @@ def test_metadata():
     assert r.status_code == 200
 
     metadata = r.json()
-    assert metadata['id'] == 'deeplab-tf'
-    assert metadata['name'] == 'deeplab TensorFlow Model'
-    assert metadata['description'] == 'deeplab TensorFlow model trained on VOCO 2012'
+    assert metadata['id'] == 'max-image-segmenter'
+    assert metadata['name'] == 'MAX Image Segmenter'
+    assert metadata['description'] == 'DeepLab TensorFlow model for semantic image segmentation, trained on VOCO 2012'
     assert metadata['license'] == 'Apache v2'
```

Comment:
Just checking: Is this `Apache v2`  our official standard value for the https://www.apache.org/licenses/LICENSE-2.0?

Reply:
We have not been consistent about this. We should standardize on something.

Reply:
https://github.ibm.com/CODAIT/Production-Readiness/issues/21